packages/cli/core/package.json

```diff
@@ -4,7 +4,7 @@
   "author": "Coveo",
   "description": "Coveo CLI",
   "engines": {
-    "node": "16.x || 18.x"
+    "node": "^16.13.0 || ^18.12.0"
```

Comment:
what are the pros/cons of specifying the minor version explicitly ?

Reply:
do we need to keep updating this number to reflect the latest LTS? Manually?

Reply:
What necessitated this change? Would be good to update PR desc with the reason.

Reply:
> what are the pros/cons of specifying the minor version explicitly ?

So, for the CLI we only support a select set of LTS(s) versions of NodeJS, currently Gallium(16) and Hydrogen(18)
The trick is that Node 18.x.x (or 16.x.x) is not necessarily an LTS version. Indeed, Node 18 became LTS with 18.12.0.
So, `18.x` was not a good semver specifier for "Hydrogen", instead we need to support:
"all versions of Node 18 with a minor equal or superior to 12." this is well managed by a caret range:
https://github.com/npm/node-semver#caret-ranges-123-025-004

> do we need to keep updating this number to reflect the latest LTS? Manually?

Only when we decide to support a new Major (next one will be 20)

> What necessitated this change? Would be good to update PR desc with the reason.

- [ ]  update desc

Reply:
Got it, thanks - it is weird though that the minor version is not mentioned in https://nodejs.dev/en/about/releases/ against LTS but then the info is mentioned in https://nodejs.org/en/blog/release/v18.12.0

In order to keep tabs on new LTS releases wonder if we can add a check in CI or a warning to dev workflow (e.g. using [node-lts-versions](https://github.com/msimerson/node-lts-versions) or [nodejs-versions](https://github.com/wallet77/nodejs-versions)) - probably as part of common monorepo tooling - so that we (dev team) will get an automated notification/warning when there is a new LTS release?

Reply:
> In order to keep tabs on new LTS releases wonder if we can add a check in CI or a warning to dev workflow (e.g. using [node-lts-versions](https://github.com/msimerson/node-lts-versions) or [nodejs-versions](https://github.com/wallet77/nodejs-versions)) - probably as part of common monorepo tooling - so that we (dev team) will get an automated notification/warning when there is a new LTS release?

I think not, for two reasons:
 - We ain't good at following up on automated alerts, due to alerts fatigue (something we would have ultimately to find a solution to)
 - https://xkcd.com/1205/  we're pretty much in the upper right corner for this IMO, so the reasonable amount of time to work on that would be 5 seconds 🤣 

Any changes to our Node-supported version will be a breaking change (at least for the foreseeable future), so it's part of a wider process.
This process should include a check on the Node release calendar so that we can foresee when our next major release will be